### DISCUSSION TITLE
xtls vision

### DISCUSSION THREAD

**Discussion by MHSanaei** [FranzKafkaYu/x-ui#207] [General]
hi
today I installed your panel I notice something
in trojan when we select tls we can't have a flow vision so on your panel there is still an option to choose it when we set tls it's not correct 
please fix it thank you

**Comment by FranzKafkaYu** [comment]
@MHSanaei if your xray version is > 1.6.1 and you choose vless,tcp and enable tls,you can m choose flow xtls-rprx-vision here,is there anything wrong?

**Comment by FranzKafkaYu** [comment]
Trojan currently won't support xtls-rprx-vision but I will suppr it finally,I add this option but you can't use it, that's it.I don't think it's a problem.
